### PR TITLE
Preparation instructions added

### PR DESCRIPTION
**Pull request by LaelLuo**

### CODE REVIEW

File: README-ZH.md [hunk 1]
```diff
@@ -54,4 +56,4 @@ Language: [English](README.md) | 中文简体
 final AssetEntity entity = await CameraPicker.pickFromCamera(context);
 ```
 
-在选择器唤起后，并且 `isAllowRecording` 为 `true`，点击拍摄按钮以拍照，长按拍摄按钮以录像。
\ No newline at end of file
+在选择器唤起后，并且 `isAllowRecording` 为 `true`，点击拍摄按钮以拍照，长按拍摄按钮以录像。
```
Comment (AlexV525) [inline README-ZH.md:59]: Why two blank lines were added?

File: README.md [hunk 2]
```diff
@@ -73,4 +75,4 @@ Reference: [wechat_assets_picker#preparing-for-use](https://github.com/flutterca
 final AssetEntity entity = await CameraPicker.pickFromCamera(context);
 ```
 
-After called and the picker displayed, plus `isAllowRecording` is `true`, tap the shooting button to take a picture, long press the shooting button to record a video.
\ No newline at end of file
+After called and the picker displayed, plus `isAllowRecording` is `true`, tap the shooting button to take a picture, long press the shooting button to record a video.
```
Comment (AlexV525) [inline README.md:78]: Same as above.

**Review by AlexV525 — CHANGES_REQUESTED** [review]
LGTM modulo comments below.

File: README-ZH.md (hunk 1 above)
Comment (LaelLuo) [reply README-ZH.md]: web ide issues

**Review by AlexV525 — APPROVED** [review]
Thanks for the first time contribution.

image

**Pull request merged**